db/main/migrate/20180420000000_constraints_and_indexes_for_gh_apps.rb

```diff
@@ -0,0 +1,17 @@
+class ConstraintsAndIndexesForGhApps < ActiveRecord::Migration[4.2]
+  self.disable_ddl_transaction!
+
+  def up
+    execute "CREATE UNIQUE INDEX CONCURRENTLY github_id_installations_idx ON installations (github_id)"
+    execute "CREATE INDEX CONCURRENTLY managed_repositories_idx ON repositories (managed_by_installation_at)"
+    execute "CREATE UNIQUE INDEX CONCURRENTLY favo_3col_uni_idx ON installations (owner_id, owner_type, removed_by_id) WHERE removed_by_id IS NOT NULL"
```

Comment:
I stole this index name from the blog, we might want to change this to match what we do with other index names. (and the one below)